authorization/token/manager/token_manager_test.go

```diff
@@ -68,8 +68,8 @@ func (s *TestTokenSuite) TestGenerateTransientUserAccessTokenForIdentity() {
 	expiresIn := time.Duration(s.Config.GetTransientTokenExpiresIn()) * time.Second
 	s.assertExpiresIn(claims["exp"], expiresIn, 10*time.Second) // 60 seconds days +/- 10 seconds
 	s.assertIntClaim(claims, "nbf", 0)
-	s.assertClaim(claims, "iss", "https://auth.openshift.io")
-	s.assertClaim(claims, "aud", "https://openshift.io")
+	s.assertClaim(claims, "iss", "http://auth.localhost")
```

Comment:
why did these expected values changed?

Reply:
@xcoulon before we used to pass request with context created using `testtoken.ContextWithRequest(nil)` this method always create request with host `https://auth.openshift.io`.  So in test the same auth_url is used.

Now, we are not using context with request and directly calling config.GetAuthServiceURL() which now will return `http://auth.localhost` and thus everywhere auth_url is now `http://auth.localhost` and openshift url is `http://localhost`.  Hope this clarifies.

Reply:
ok, makes sense. Thanks for the clarification @nurali-techie